datafusion/sqllogictest/test_files/aal.slt

```diff
@@ -0,0 +1,232 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
this file probably at least needs to be renamed to topk.slt or something other than my initials (`AAL` 😆 )

I am also not sure it is the best test of the topk coverage -- I had this for quickly iterating during development

Reply:
Aha I was wondering what `aal` was standing for 🤣